project/Dependencies.scala

```diff
@@ -24,7 +24,6 @@ object Dependencies {
     // when updating config version, update links ActorSystem ScalaDoc to link to the updated version
     val config        = "com.typesafe"                % "config"                       % "1.2.1"       // ApacheV2
     val netty         = "io.netty"                    % "netty"                        % "3.8.0.Final" // ApacheV2
-    val protobuf      = "com.google.protobuf"         % "protobuf-java"                % "2.5.0"       // New BSD
```

Comment:
yes, but those who need this will be happy to workaround it by adding the manual exclude
for 2.4 I don't think we have to be that strict